src/seq/mod.rs

```diff
@@ -265,7 +265,8 @@ pub trait SliceRandom {
 
 /// Extension trait on iterators, providing random sampling methods.
 ///
-/// This trait is implemented on all sized iterators, providing methods for
+/// This trait is implemented on all iterators `I` where `I: Iterator + Sized`
+/// and provides methods for
 /// choosing one or more elements. You must `use` this trait:
```

Comment:
Those lines should probably be joined?

Reply:
Markdown considers them the same paragraph. There's not really any point in re-wrapping this I think?

Reply:
I think it improves readability, but I guess that's subjective. :shrug: